src/VoteDelegate.sol

```diff
@@ -95,6 +104,11 @@ contract VoteDelegate {
         emit Free(msg.sender, wad);
     }
 
+    function reserveHatch() external {
+        require(block.number > hatchTrigger + HATCH_SIZE + HATCH_COOLDOWN, "VoteDelegate/cooldown-not-finished");
```

Comment:
imo a "reasonable" mev bot will never implement a code to call this function. Especially when the upside of `bark` is very limited.

An ugly way to educate mev bots about it, is to force it on `bark`. Meaning `bark` will fail if `reserveHatch` was not previously called (in the last X blocks)